crates/holochain/src/core/ribosome/guest_callback/init.rs

```diff
@@ -292,4 +302,75 @@ mod slow_tests {
             InitResult::Fail(TestWasm::InitFail.into(), "because i said so".into()),
         );
     }
+
+    #[tokio::test(flavor = "multi_thread")]
+    async fn conductor_will_not_accept_zome_calls_before_the_network_is_initialised() {
+        let (dna_file, _, _) = SweetDnaFile::from_test_wasms(
+            random_network_seed(),
+            vec![TestWasm::Create],
+            SerializedBytes::default(),
+        )
+            .await;
+
+        let mut conductor = SweetConductor::from_standard_config().await;
+
+        let app = conductor.setup_app("app", [&dna_file]).await.unwrap();
+
+        let cloned = conductor.create_clone_cell(CreateCloneCellPayload {
+            app_id: app.installed_app_id().clone(),
+            role_name: dna_file.dna_hash().to_string().clone(),
+            modifiers: DnaModifiersOpt::none().with_network_seed("anything else".to_string()),
+            membrane_proof: None,
+            name: Some("cloned".to_string()),
+        }).await.unwrap();
+
+        let enable_or_disable_payload = DisableCloneCellPayload {
+            app_id: app.installed_app_id().clone(),
+            clone_cell_id: CloneCellId::CloneId(cloned.clone_id.clone()),
+        };
+        conductor.disable_clone_cell(&enable_or_disable_payload).await.unwrap();
+
+        let zome: SweetZome = SweetZome::new(
+            cloned.cell_id.clone(),
+            TestWasm::Create.coordinator_zome_name(),
+        );
+
+        // Run the cell enable in parallel. If we wait for it then we shouldn't see the error we're looking for
+        let conductor_handle = conductor.raw_handle().clone();
+        tokio::spawn(async move {
+            conductor_handle.enable_clone_cell(&enable_or_disable_payload).await.unwrap();
+        });
+
+        // conductor.raw_handle().enable_clone_cell(&enable_or_disable_payload).await.unwrap();
```

Comment:
I will remove this, but in case anybody wants to look at what this test is doing while the PR is open. Removing this line amd commenting out the spawned task above causes the test to fail